### PR TITLE
Allow to disable Option conversion for output parameters and bool ret…

### PR DESCRIPTION
**Pull request by EPashkin**
…urn type
Part of https://github.com/gtk-rs/pango/issues/62

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Please take look, now PR includes forgotten config parameters.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ name = "Gtk.SomeClass"
 status = "generate"
 # replace the parameter name for the child in child properties (instead "child")
 child_name = "item"
+# force trait SomeClassExt generation even if class has no children
```
Comment (GuillaumeGomez) [inline README.md]: Nice to keep it up to date! :)

File: README.md [hunk 2]
```diff
@@ -53,6 +53,12 @@ name = "Gtk.SomeClass"
...
+trait = true
+# allow rename result file
+module_name = "soome_class"
```
Comment (GuillaumeGomez) [inline README.md]: "soome"? ;)

**Comment by GuillaumeGomez** [comment]
Just one nit and it's good for me.

File: README.md (hunk 2 above)
Comment (EPashkin) [inline README.md]: Yes to make difference visible.

Comment (GuillaumeGomez) [inline README.md]: Ah I see!

Comment (EPashkin) [inline README.md]: Maybe you have better name?

Comment (GuillaumeGomez) [inline README.md]: No, it's fine as is.

**Pull request merged**